pico_w/bt/standalone/server.c

```diff
@@ -74,20 +87,19 @@ int main() {
     // turn on bluetooth!
     hci_power_control(HCI_POWER_ON);
     
-    // btstack_run_loop_execute is only required when using the 'polling' method (e.g. using pico_cyw43_arch_poll library).
-    // This example uses the 'threadsafe background` method, where BT work is handled in a low priority IRQ, so it
-    // is fine to call bt_stack_run_loop_execute() but equally you can continue executing user code.
-
-#if 0 // btstack_run_loop_execute() is not required, so lets not use it
-    btstack_run_loop_execute();
-#else
-    // this core is free to do it's own stuff except when using 'polling' method (in which case you should use 
-    // btstacK_run_loop_ methods to add work to the run loop.
-    
-    // this is a forever loop in place of where user code would go.
-    while(true) {      
+    key_pressed = false;
+    while(!key_pressed) {
```

Comment:
tiny nitpick: the lack-of-space after `while` and before `(` (and also on line 100) is inconsistent with the `if (` on line 56

Reply:
I literally never put a space after while in my whole programming career.